audits/audit.js

```diff
@@ -39,16 +39,25 @@ class Audit {
   }
 
   /**
+   * @return {boolean|number|string|undefined}
+   */
+  static get optimalValue() {
+    return undefined;
+  }
+
+  /**
    * @param {(boolean|number|string)} value
    * @param {?(boolean|number|string)=} rawValue
    * @param {string=} debugString Optional string to describe any error condition encountered.
+   * @param {?(boolean|number|string)=} optimalValue
    * @return {!AuditResult}
    */
-  static generateAuditResult(value, rawValue, debugString) {
+  static generateAuditResult(value, rawValue, debugString, optimalValue) {
```

Comment:
Yeah, I think this is going to get a boost at some point, because we're likely to want to expose some kind of optimalValue as an object with a bunch of suggestions, i.e. "gzip these and get Xms improvement, and utilize Y% less bandwidth"